monster-hunter-world.asl

```diff
@@ -36,7 +36,7 @@ state("MonsterHunterWorld", "421471") {
     float sessionTime: "MonsterHunterWorld.exe",0x0506F240,0x90,0x46A0,0x14;
```

Comment:
Update this line!

Needs change sessionTime pointer.
Check This [CE table](https://opencheattables.org/viewtopic.php?f=12&t=20&p=31&hilit=monster+hunter+world#p31)